### PR TITLE
docs: add harbor, update screenshots

### PR DESCRIPTION
**Pull request by caroldelwing**
## Describe the Change

This PR updates the Deploy a Custom Pack tutorial. 
- Add Harbor as an example of Basic OCI registry.
- Update the screenshots.
- Add Podman as a container engine option.

## Review Changes

💻 [Preview URL](https://deploy-preview-2133--docs-spectrocloud.netlify.app/registries-and-packs/deploy-pack)

🎫 [DOC-1041](https://spectrocloud.atlassian.net/browse/DOC-1041)
🎫 [DOC-1007](https://spectrocloud.atlassian.net/browse/DOC-1007)

### CODE REVIEW

**Review by lennessyy — APPROVED** [review]
Looks great! I literally couldn't find anything

**Review by yuliiiah — APPROVED** [review]
Hey @caroldelwing, I've also taken a look, and agree with Lenny, great work! I left one small idea that's completely up to you :)

File: docs/docs-content/registries-and-packs/deploy-pack.md [hunk 1]
```diff
@@ -631,25 +639,23 @@ Now, proceed to create the pack repository and push the Hello Universe pack to t
 oras push $HARBOR_ADDRESS/$HARBOR_PROJECT/spectro-packs/archive/$NAME:$VERSION $NAME-$VERSION.tar.gz
 ```
 
-To confirm the presence of the pack in the repository, open your web browser, access your Harbor address, click on **Projects**, select the **spectro-oci-registry** project, and then click on the **spectro-oci-registry/spectro-packs/archive/hellouniverse** repository. You should see the pushed artifact, as shown in the image below.
+To confirm the presence of the pack in the repository, open your web browser, access your Harbor address, click on
+**Projects**, select the **spectro-oci-registry** project, and then click on the
+**spectro-oci-registry/spectro-packs/archive/hellouniverse** repository. You should see the pushed artifact, as shown in
```
[the remark this answers is not in the document]
Comment (karl-cardenas-coding) [reply docs/docs-content/registries-and-packs/deploy-pack.md:644]: @caroldelwing Vale raised a good point here with the word "see" 😉

[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/registries-and-packs/deploy-pack.md:644]: Oh yes, definitely. I missed this one :sweat_smile:

File: docs/docs-content/registries-and-packs/deploy-pack.md [hunk 2]
```diff
@@ -80,12 +80,16 @@ infrastructure to avoid additional charges.
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/registries-and-packs/deploy-pack.md:80]: Hi there :) Thank you for your feedback. It is super valid. We recommend the infra in the _Create a Cluster_ section (instance type, disk size, min number of CPU cores). Users can also access an estimated hourly cost (_Create a Cluster - Review_ section). I'll update the paragraph to say that there are cloud expenses associated.

**Pull request merged**